plugins/repository-s3/src/yamlRestTest/resources/rest-api-spec/test/repository_s3/60_repository_eks_credentials.yml

```diff
@@ -0,0 +1,268 @@
+# Integration tests for repository-s3
+
+---
+setup:
+
+  # Register repository with eks credentials
+  - do:
+      snapshot.create_repository:
+        repository: repository_eks
+        body:
+          type: s3
+          settings:
+            bucket: ${eks_bucket}
+            client: integration_test_eks
+            base_path: "${eks_base_path}"
+            canned_acl: private
+            storage_class: standard
+            disable_chunked_encoding: ${disable_chunked_encoding}
+
+---
+"Snapshot and Restore with repository-s3 using eks credentials":
```

Comment:
I think you're just following the established pattern here, but credentials are the kind of thing that will either work or not, so is it necessary to duplicate the full test scenario for different credentials? It seems to me we'd get similar coverage by running the full test once, and then just doing a minimal test for each credential type. Thoughts? I don't want to block on this though.

Reply:
Yeah, this is just a pattern here, essentially I copy/pasted other test case(s). To be fair, duplication is / was the easiest way to come up with the test suite for the new flow. All test cases are looking uniform in this regards, but to your point - indeed the creds are either present or not, one scenario should be sufficient (keeping suites the same for all flows eliminate the question why one has these steps but other doesn't).